src/interfaces/coral_web/src/hooks/startModes.tsx

```diff
@@ -1,4 +1,4 @@
-import { DEFAULT_CHAT_TOOL } from '@/cohere-client';
+import {DEFAULT_CHAT_TOOL, DEFAULT_ELASTIC_SEARCH_TOOL} from '@/cohere-client';
```

Comment:
Would you mind removing the FE changes? Since the community features are not required to run the toolkit, this could lead to errors or confusing UX.